tests/test_ocean_cli.py

```diff
@@ -23,6 +24,11 @@ def test_ocean_cli():
     program = sys.argv[0]
     if program[0] != '/':
         program = os.path.join(os.getcwd(), program)
+
+    # fix for running from the command line 'python -m pytest tests' within venv
+    if re.search(r'/venv/', program):
+        program = './'
```

Comment:
I'm a little concerned about this doing the right thing depending upon how the tests are invoked.
As I cannot test it now I will assume it's correct.

Reply:
I apologize , it's a quick fix to get the tests running under venv workng